### PR TITLE
(MAINT) Fix package tests

### PR DESCRIPTION
**Pull request by chelnak**
This changes includes fixes for the post packaging smoke tests.

### CODE REVIEW

File: package-testing/spec/package/update_module_spec.rb [hunk 1]
```diff
@@ -3,15 +3,13 @@
 require 'json'
 
 modules = [
-  'puppetlabs/puppetlabs-motd',
-  'puppetlabs/puppetlabs-concat',
-  'puppetlabs/puppetlabs-inifile'
+  'puppetlabs/puppetlabs-motd'
```
Comment (GSPatton) [inline package-testing/spec/package/update_module_spec.rb:6]: what's the rationale behind only testing against motd?

File: package-testing/spec/test_hosts.yml
Comment (GSPatton) [inline package-testing/spec/test_hosts.yml:1]: is this needed?

Comment (chelnak) [reply package-testing/spec/test_hosts.yml:1]: Good spot

File: package-testing/spec/package/update_module_spec.rb (hunk 1 above)
Comment (chelnak) [reply package-testing/spec/package/update_module_spec.rb:6]: It reduces the surface of issues. MOTD changes are less frequent than the others.

**Review by GSPatton — APPROVED** [review]

**Pull request merged**